openid-4-verifiable-presentations-1_0.md

```diff
@@ -1516,9 +1516,9 @@ The Verifier can use one of the following mechanisms to invoke a Wallet:
 - Custom URL scheme as an `authorization_endpoint` (for example, `openid4vp://` as defined in (#openid4vp-scheme))
 - URL (including Domain-bound Universal Links/App link) as an `authorization_endpoint`
 
-For a cross device flow, either of the above options MAY be presented as a QR code for the End-User to scan using a wallet or an arbitrary camera application on a user-device.
+For a cross device flow, either of the above options MAY be presented as a QR code for the End-User to scan using a Wallet or an arbitrary camera application on a user-device.
 
-The Wallet can also be invoked from the web or a native app using the Digital Credentials API as described in (#dc_api). As described in detail in (#dc_api), DC API provides privacy, security (see (#session_fixation)), and user experience benefits (particularly in the cases where a user has multiple Wallets).
+The Wallet can also be invoked from the web or a native app using the Digital Credentials API as described in (#dc_api). As described in detail in (#dc_api), DC API provides privacy, security (see (#session_fixation)), and user experience benefits (particularly in the cases where an End-User has multiple Wallets).
```

Comment:
Seems like we are filling an inconsistency here by replacing End-User Authentication in the diagram about with User Authentication and then changing this from user to End-User. Any objection to just using End-User consistently?

Reply:
it's actually very intentional. in general we use "end-user" but there are places, where end-user does not make sense. like, we don't say end-user authorization, end-user authentication, etc. we just say user authentication and user authorization. and i don't want to artificially say end-user authentication in this spec.